app/code/OleksiiBezpoiasnyi/ControllerDemo/view/frontend/templates/info.phtml

```diff
@@ -0,0 +1,4 @@
+<?php /** @var \OleksiiBezpoiasnyi\ControllerDemo\Block\Demo $block */ ?>
+<?php /** @var \Magento\Framework\Escaper $escaper */ ?>
+<p class="oleksiib-controller-demo"><?= $escaper->escapeHtml(__('Full name: %1', $block->getFirstName() . ' ' . $block->getLastName())) ?></p>
+<p class="oleksiib-controller-demo"><?= $escaper->escapeHtml(__('Repository URL: ')) . '<a href="' . $block->getRepositoryUrl() . '">' . $escaper->escapeUrl($block->getRepositoryUrl()) ?></a></p>
```

Comment:
@AlexeyBezpoyasny Incorrect escaping:
1) `href` must be escaped with `escapeUrl`
2) link text must be escaped with `escapeHtml`